example_problems/boolfind/output_visualizer/boolfind_visual/visualize.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+#
+# Invoke as:
+# <output_visualizer_program> answer_file feedback_file
+import matplotlib.pyplot as plt
+import sys
+
+my_name  = sys.argv[0]
+my_input = sys.argv[1]
+my_feedback = sys.argv[2]
+
+with open(my_input, 'r') as f:
+  lines = f.readlines()
+  vals = []
+  for line in lines:
+    if 'READ' in line:
+      vals.append(int(line.split(' ')[-1]))
```

Comment:
This breaks if the submission writes bogus output, e.g. `READ domjudge`, how do we handle cases like this?

Reply:
I would say this is something which is left up to the jury.

I wonder if there is something we can do with the output_validator & the judgemessage for communication. Either an extra exitcode or otherwise the jury has to parse the output in this script.

Reply:
Yes, it's left to the jury. In this case you are the jury by implementing the visualizer :-)

Since people are going to model after examples we give them, we should make it robust and not crash.

Reply:
Adding to this, I think we should return 42 in the script if we were able to visualize the output, and 43 if not.